examples/http/helloworld_client/main.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"log"
+	"net/http"
+	"time"
+
+	"go.opencensus.io/plugin/ochttp"
+	"go.opencensus.io/trace"
+
+	"go.opencensus.io/examples/exporter"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/zpages"
+)
+
+const server = "http://localhost:50030"
+
+func main() {
+	go func() { log.Fatal(http.ListenAndServe(":8080", zpages.Handler)) }()
+
+	// Register stats and trace exporters to export the collected data.
+	exporter := &exporter.Exporter{}
+	view.RegisterExporter(exporter)
+	trace.RegisterExporter(exporter)
+
+	view.SetReportingPeriod(1 * time.Second)
+
+	client := &http.Client{
+		Transport: &ochttp.Transport{},
+	}
+
+	for {
```

Comment:
Please remove this for loop, this is a tight loop, why not just one get
```go
res, err := client.Get(url)
...
```

Reply:
Done.